content/millennium/r4/diagnostic/observation.md

```diff
@@ -124,4 +126,5 @@ The common [errors] and [OperationOutcomes] may be returned.
 [errors]: ../../#client-errors
 [OperationOutcomes]: ../../#operation-outcomes
 [Extensions for converting between versions]: https://www.hl7.org/fhir/r4/versions.html#extensions
-[R5 Snapshot of Observation.value]: https://hl7.org/fhir/2020Feb/observation-definitions.html#Observation.value_x_
\ No newline at end of file
+[R5 Snapshot of Observation.value]: https://hl7.org/fhir/2020Feb/observation-definitions.html#Observation.value_x_
```

Comment:
it doesn't look like line 129 changed, any idea what is being highlighted by this diff?

Reply:
I think that is because I pressed enter to go to next line, seems fine as URL is still same